pkg/translator/prometheusremotewrite/helper_test.go

```diff
@@ -425,6 +425,16 @@ func Test_getPromExemplars(t *testing.T) {
 				},
 			},
 		},
+		{
+			"too_many_characters_drops_labels",
```

Comment:
Maybe add a test here where rune count is less than byte count and byte count is >128.

Reply:
done